src/pages/react/training/react-fundamentals/london/index.js

```diff
@@ -0,0 +1,177 @@
+            title="React Fundamentals in London"
+            meta={[
+              {
+                name: 'description',
+                content:
+                  'React Fundamentals - learn a solid foundation in React in just 3 days in London',
+              },
+            ]}
+          />
+          <Breadcrumb
+            path={[
+              { to: '/', label: 'Home' },
+              { to: '/react', label: 'React' },
+              { to: '/react/training/', label: 'Training' },
+              {
+                to: '/react/training/react-fundamentals',
+                label: 'React Fundamentals',
+              },
+              {
+                to: '/react/training/react-fundamentals/london',
+                label: 'London',
+              },
+            ]}
+          />
+          <Header
+            titleLines={['React, Redux Fundamentals - London']}
+            subtitle="In 3 days, we can teach you how to code in React to give you a solid foundation to take your dev career further."
+            links={header.landingTraining.links}
+            bgImageName={BOOTCAMP}
+            type={REACT_FUNDAMENTALS}
+            training={training}
+            showInfoBox={true}
+          />
+          <TopSection xsBgDark top>
+            <Grid>
+              <Card bg="dark">
+                <Row>
+                  <Col md={6} lg={5} lgOffset={1}>
+                    <PaymentSection
+                      training={training}
+                      trainingError={trainingError}
+                      trainingLoading={trainingLoading}
+                    />
+                  </Col>
+                  <Col md={6} lg={4} lgOffset={1}>
+                    <Video youtubeId="yvROXLQ1jHg" />
+                    <TrainingDetails coaches={[ALEX_LOBERA, RICHARD_MOSS]} />
+                  </Col>
+                </Row>
+              </Card>
+            </Grid>
+          </TopSection>
+          <Section>
+            <Grid>
+              <Row>
+                <HideComponentsUsingCss xs sm>
+                  <Col md={6} lg={5}>
+                    <Image src={BOOTCAMP_COLLAB} width="100%" />
+                  </Col>
+                </HideComponentsUsingCss>
+                <Col md={6} lg={5} lgOffset={1}>
+                  <H2Ref>
+                    Is this React course right for you? Are you...{' '}
+                    <Link to="#target-audience" name="target-audience">
+                      #
+                    </Link>
+                  </H2Ref>
+                  <Ul>
+                    <Li>
+                      A developer with 1+ year of development under your belt
+                      using JavaScript?
+                    </Li>
+                    <Li>
+                      Familiar with front-end technologies like JavaScript, CSS,
+                      and HTML?
+                    </Li>
+                    <Li>
+                      Taking a step forward to become a React JS Specialist able
```

Comment:
@ric9176 I think this is for the advanced, not for the fundamentals

Reply:
sorted :-)